crates/sui-rpc-api/proto/sui/rpc/v2alpha/transaction_execution_service.proto

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package sui.rpc.v2alpha;
+
+import "google/protobuf/empty.proto";
+import "google/protobuf/field_mask.proto";
+import "sui/rpc/v2alpha/executed_transaction.proto";
+import "sui/rpc/v2alpha/signature.proto";
+import "sui/rpc/v2alpha/transaction.proto";
+
+service TransactionExecutionService {
+  rpc ExecuteTransaction(ExecuteTransactionRequest) returns (ExecuteTransactionResponse);
+}
+
+message ExecuteTransactionRequest {
+  // The transaction to execute.
+  // Do we want to have this only be BCS?
```

Comment:
I think it's fine to have a structured version of this API, but the absence of an API that accepts the transaction in its raw BCS form is a bit strange.